flutter_local_notifications/lib/src/platform_specifics/android/status_bar_notification.dart

```diff
@@ -0,0 +1,22 @@
+class ActiveNotification {
+  const ActiveNotification(
+    this.id,
+    this.channelId,
+    this.title,
+    this.body,
+  );
+
+  final int id;
+  final String channelId;
+  final String title;
+  final String body;
+
+  toString() {
```

Comment:
Can this be removed? Haven't implemented this for any other class in the plugin and removing it would allow users of the plugin to define their own implementation of `toString` e.g. via extension methods?

Reply:
sure